modules/setting/setting.go

```diff
@@ -478,6 +478,10 @@ func getWorkPath(appPath string) string {
 			workPath = appPath[:i]
 		}
 	}
+	workPath = strings.ReplaceAll(workPath, "\\", "/")
```

Comment:
`filepath.Join` can deal with backslashes on Windows. It doesn't need such hacks. After all, backslash is a valid path character on many file sytems.

Reply:
This is to be consistent with the rest of the filepath munging in this file.